taggit/managers.py

```diff
@@ -365,8 +365,13 @@ def similar_objects(self):
             objs = rel_model._default_manager.filter(**{
                 "%s__in" % remote_field.field_name: [r["content_object"] for r in qs]
             })
+            actual_remote_field_name = remote_field.field_name
+            if VERSION > (1, 9):
```

Comment:
taggit no long supports Django < 1.11. Could you update the PR to drop workarounds for older Djangos?